cmd/dlt/logforwarder/cmd.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logforwarder
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "log-forwarder -c <cluster-id> <log-forwarder-id>"
+	short   = "Delete log forwarder"
+	long    = "Delete a log forwarder from a cluster."
+	example = `  # Delete log forwarder with ID 'example-id' from a cluster named 'mycluster-hcp'
+  rosa delete log-forwarder --cluster=mycluster-hcp example-id`
+)
+
+var (
+	aliases = []string{"log_forwarder", "logforwarder"}
```

Comment:
I see `log-forwarder` in the use/examples above. Should this match?

Reply:
It doesn't need to- but it is a bit misleading so I went ahead and added it anyway

the examples above I ran with `log-forwarder` as the subcommand 🙂